src/Stratis.Bitcoin.Features.PoA/Voting/Poll.cs

```diff
@@ -32,6 +32,12 @@ public Poll()
         /// </summary>
         public bool IsPending => this.PollVotedInFavorBlockData == null;
 
+        /// <summary><c>true</c> if poll has expired; <c>false</c> otherwise.</summary>
```

Comment:
```suggestion
        /// <summary><c>true</c> if the poll's `PollVotedInFavorBlockData` is NOT null and it's height set to 0 via it's HashHeighPair; <c>false</c> otherwise.
        /// When a poll is regarded as `Expired` its means that it's `PollVotedInFavorBlockData` was set to a HashHeighPair of 1 (hash) and 0 (height) in the VotingManager's ProcessBlock method.
        </summary>
```

Reply:
It may be sufficient to note that a value of `0` for the `height` field alone flags a poll as "Expired". Note that the value of the `hash` is disregarded in this regards and only matters when serializing the field as a whole. As such commenting on the hash is somewhat counter-productive in this context. Also commenting on where the field is being set is of fleeting interest and may not remain accurate.

Reply:
I've added the following remark to this comment:
```
/// <remarks>A poll is flagged as "Expired" by setting <see cref="Poll.PollVotedInFavorBlockData.Height"/> to zero.</remarks>
```